Workaround for installing 1.7.0 on Windows

Installing closure-util 1.7.0 on Windows failed for me. I did some testing to rule out if it was my environment and came up empty handed. Eventually I came up with the subsequent workaround.

To test, I created an empty directory, `testdir`, and created the following `package.json` in it:
```json
{
  "private": true,
   "devDependencies": {
     "closure-util": "1.7.0"
   }
}
```

Running `npm install` from testdir results in the following (snipped for brevity):

```
$ npm install

> ws@0.4.32 install C:\testdir\node_modules\closure-util\node_modules\socket.io\node_modules\socket.io-client\node_modules\ws
...snip...

> closure-util@1.7.0 postinstall C:\testdir\node_modules\closure-util
> node ./bin/closure-util.js update

module.js:338
    throw err;
          ^
Error: Cannot find module 'temp'
    at Function.Module._resolveFilename (module.js:336:15)
...snip...
```
I guessed that the `temp` module should have been a regular dependency instead of a devDependency for installing closure-util with npm. I found that modifying the `package.json` for closure-util in my npm-cache (`%appdata%\npm-cache\closure-util\1.7.0\package\package.json` in my case) by moving the `"temp":"^0.8.1"` portion from the `"devDependencies"` section to the `"dependencies"` section permitted the `npm install` command to proceed as normal afterward:

```
$ npm install
...snip...
> closure-util@1.7.0 postinstall C:\testdir\node_modules\closure-util
> node ./bin/closure-util.js update

info install Downloading http://dl.google.com/closure-compiler/compiler-20150729.zip
info install Downloading https://github.com/google/closure-library/archive/0011afd534469ba111786fe68300a634e08a4d80.zip
info install Received 72% (4041K)
info install Received 80% (4479K)
...snip...
```

Suggesting that the `temp` dependency move from devDependencies to dependencies going forward.

--- a/package.json
+++ b/package.json
@@ -28,7 +28,8 @@
     "npmlog": "0.0.6",
     "rimraf": "2.2.8",
     "send": "0.12.2",
-    "socket.io": "0.9.16"
+    "socket.io": "0.9.16",
+    "temp": "^0.8.1"
   },
   "devDependencies": {
     "chai": "~1.8.1",
@@ -37,8 +38,7 @@
     "grunt-contrib-jshint": "~0.8.0",
     "grunt-contrib-watch": "~0.6.1",
     "grunt-mocha-test": "~0.8.1",
-    "supertest": "^0.15.0",
-    "temp": "^0.8.1"
+    "supertest": "^0.15.0"
   },
   "scripts": {
     "postinstall": "node ./bin/closure-util.js update",

